utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -64,7 +64,8 @@ int main(int argc, char **argv)
     int decoderType = atoi(argv[13]);
     int batchSize = atoi(argv[14]);
 
-    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 79);
+    bool additionalParamCase = (testCase == 8 || testCase == 21 || testCase == 23 || testCase == 24 || testCase == 49 || testCase == 79);
```

Comment:
we need to remove all these hardcoded testCase manipulation and use more standard coding methods like string or map table. I guess this should be done in a separate PR